src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -38,6 +38,23 @@ object ExportVCF {
     }
   }
 
+  def emitGenotype(f: Field, sb: StringBuilder, value: Annotation) {
+    if (value == null)
+      sb += '.'
+    else
+      f.typ match {
+        case it: TIterable =>
+          val arr = value.asInstanceOf[Iterable[_]]
+          if (arr.isEmpty) {
+            sb += '.'
```

Comment:
Shouldn't an empty list be empty, not missing?  What does VCF spec say?  Hahahahha.